src/components/NFTPage/CollectorList/index.vue

```diff
@@ -26,37 +55,39 @@
           v-if="shouldShowMore"
           @click-show-more="$emit('click-show-more-collector')"
         >
-          <template #header>
-            <div class="flex justify-between mb-[12px] min-w-[310px]">
-              <Label
-                class="w-min font-600"
-                :text="
-                  `${$t('nft_details_page_title_collector')} (${items.length})`
-                "
-                preset="h5"
-                valign="middle"
-                content-class="whitespace-nowrap text-like-green "
-                prepend-class="text-like-green"
-              >
-                <template #prepend>
-                  <IconPerson />
-                </template>
-              </Label>
-              <Label
-                class="w-min font-600"
-                :text="$t('nft_details_page_label_owning')"
-                preset="h5"
-                valign="middle"
-                content-class="whitespace-nowrap text-like-green "
-                prepend-class="text-like-green"
-              />
-            </div>
-            <hr class="w-full border-shade-gray" />
-          </template>
```

Comment:
Why remove this?

Reply:
I wanted to hide the header like the history modal, but let's align the UI maybe later ~